src/main/java/com/esri/core/geometry/WkbExportFlags.java

```diff
@@ -37,5 +37,7 @@
 	public static final int wkbExportMultiPolygon = 32;
 	public static final int wkbExportStripZs = 64;
 	public static final int wkbExportStripMs = 128;
+	public static final int wkbExportAsExtendedWkb = 256;
+	public static final int wkbExportStripSrid = 512;
```

Comment:
This flag should be renamed to indicate it is for the extended WKB.

Reply:
so `ewkbExportStripSrid`?

Reply:
okay

Reply:
https://github.com/Esri/geometry-api-java/commit/b0715e090d6c2b2bc20dff4636549a5ebc6e3ccc#diff-50a693fe943d8766d4616bba6ac739b38613a585dd3463155726c406e6100326R40